modules.yml

```diff
@@ -781,6 +781,16 @@
   - censysio_id
   - censysio_secret
   version: 2.0
+- author: Qazeer
+  dependencies: []
+  description: Uses the HackerTarget.com reverse IP lookup API to find host names
+    associated with IP addresses. Updates the 'hosts' table with the results.
+  files: []
+  last_updated: '2021-10-04'
+  name: HackerTarget reverse IP lookup
+  path: recon/hosts-hosts/hackertarget_reverse
+  required_keys: []
```

Comment:
Even though it technically isn't required, please add the key name here because this is what tells the framework to create a slot for it in the key store.